packages/blueprints-integration/src/context/onSetAsNextContext.ts

```diff
@@ -76,5 +76,5 @@ export interface IOnSetAsNextContext extends IShowStyleUserContext, IEventContex
 	 * Multiple calls of this inside one call to `onSetAsNext` will replace earlier calls.
 	 * @returns Whether a new Part was found using the provided offset
 	 */
-	moveNextPart(partDelta: number, segmentDelta: number): Promise<boolean>
+	moveNextPart(partDelta: number, segmentDelta: number, ignoreQuickLoop: boolean): Promise<boolean>
```

Comment:
the parameter is optional in other places, including the implementation, should it be optional here too?